pyproject.toml

```diff
@@ -44,6 +44,7 @@ requests = "*"
 pre-commit = "*"
 hypothesis = "*"
 mkdocs-gen-files = "*"
+types-python-jose = "*"
```

Comment:
you can install this as part of the `.pre-commit-config.yaml` by adding it to the mypy `additional_dependencies` - unless its required here.